.travis.yml

```diff
@@ -7,3 +8,18 @@ before_script: bundle update
 script: "script/cibuild"
 sudo: false
 cache: bundler
+
+matrix:
+  exclude:
+    - rvm: 1.9.3
+      env: JEKYLL_VERSION=3.0.0.beta8
+    - env: JEKYLL_VERSION=2.4
+      rvm: 2.1
+    - rvm: 2.2
+      env: JEKYLL_VERSION=2.4
```

Comment:
Honestly, this was copy&paste from @envygeeks work on jekyll/jekyll-feed#53

I can include them and see what breaks?